.github/workflows/bun-test.yml

```diff
@@ -13,6 +13,10 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      -name: set environment variables
+       env:
+         TSCIRCUIT_TESTS_AUTH: true
+
```

Comment:
```suggestion
```

Reply:
in `getTestServer` you'll want to do something like `process.env.TSCIRCUIT_AUTH_TOKEN = "testtoken"`